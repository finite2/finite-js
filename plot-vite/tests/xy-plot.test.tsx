import * as XYPlotExamples from "examples/xy-plot";
import { renderTest } from "./test-utils";

describe("XYPlot example render", () => {
  renderTest(XYPlotExamples);
});
