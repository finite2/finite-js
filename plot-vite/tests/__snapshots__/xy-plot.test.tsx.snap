// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`XYPlot example render > XYPlotExample0 renders 1`] = `
<div
  id="test-id"
>
  <svg
    class="user-select-none relative"
    viewBox="0 0 900 600"
  >
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="550"
          width="840"
          x="50"
          y="10"
        />
      </g>
    </g>
  </svg>
</div>
`;

exports[`XYPlot example render > XYPlotExample1 renders 1`] = `
<div
  id="test-id"
>
  <svg
    class="user-select-none relative"
    viewBox="0 0 900 600"
  >
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="390"
          x="50"
          y="10"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="390"
          x="50"
          y="310"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="390"
          x="500"
          y="10"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="390"
          x="500"
          y="310"
        />
      </g>
    </g>
  </svg>
</div>
`;

exports[`XYPlot example render > XYPlotExample2 renders 1`] = `
<div
  id="test-id"
>
  <svg
    class="user-select-none relative"
    viewBox="0 0 900 600"
  >
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="540"
          x="50"
          y="10"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="240"
          x="650"
          y="10"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="240"
          x="50"
          y="310"
        />
      </g>
    </g>
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="250"
          width="540"
          x="350"
          y="310"
        />
      </g>
    </g>
  </svg>
</div>
`;

exports[`XYPlot example render > XYPlotExample3 renders 1`] = `
<div
  id="test-id"
>
  <svg
    class="user-select-none relative"
    viewBox="0 0 900 600"
  >
    <g
      class="plot__xyplot"
    >
      <g
        class="plot__region"
      >
        <rect
          class="plot__region-dragcatcher"
          fill="var(--color-background-alt)"
          height="510"
          width="810"
          x="80"
          y="10"
        />
      </g>
    </g>
  </svg>
</div>
`;
